solr/core/src/java/org/apache/solr/handler/component/HttpShardHandler.java

```diff
@@ -191,7 +244,7 @@ protected QueryRequest makeQueryRequest(
   }
 
   /** Subclasses could modify the Response based on the shard */
-  protected ShardResponse transfomResponse(
+  protected ShardResponse transformResponse(
```

Comment:
I merged a PR that corrects that this wasn't being called above -- 391cdd04277b83a010f1d33288f9bc344a7ba2d0